tests/integration/install-kind.sh

```diff
@@ -59,14 +59,13 @@ function ensure_kind_network() {
   fi
 }
 
-function install_cloud_provider_kind() {
-  # https://github.com/kubernetes-sigs/cloud-provider-kind
-  if [[ ! -d cloud-provider-kind ]]; then
-    git clone --depth 1 https://github.com/kubernetes-sigs/cloud-provider-kind
-  fi
-  pushd cloud-provider-kind
-  sudo bash -c 'NET_MODE=kind docker compose up -d'
-  popd
+function run_cloud_provider_kind() {
+  docker run -d \
+    --network kind \
+    --restart unless-stopped \
+    --privileged \
```

Comment:
Minor point but is `--privileged` a specific to get it to run with WSL (and could be removed in the CI)?

Reply:
Fair point - it's not included in the CPK compose.yml

Reply:
Good catch!